src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2919,6 +2914,23 @@ private BoundExpression BindDelegateCreationExpression(ObjectCreationExpressionS
                     return new BoundDelegateCreationExpression(node, boundLambda, methodOpt: null, isExtensionMethod: false, type: type, hasErrors: !conversion.IsImplicit);
                 }
 
+                else if (analyzedArguments.HasErrors)
+                {
+                    // There is no hope, skip.
+                }
+
+                // 2. A method group
+                else if (argument.Kind == BoundKind.MethodGroup)
```

Comment:
Why was this case moved down? Is the argument `Kind` for an `UnboundLambda` also `BoundKind.MethodGroup`?